src/bluesky/callbacks/mpl_plotting.py

```diff
@@ -469,6 +472,7 @@ def __init__(
         ax=None,
         x_positive="right",
         y_positive="up",
+        title="",
```

Comment:
maybe default to `None` so that the user can pass an empty string to suppress the title?